RecoLocalTracker/Configuration/python/RecoLocalTracker_Cosmics_cff.py

```diff
@@ -8,7 +8,9 @@
 from RecoLocalTracker.SiPixelClusterizer.SiPixelClusterizer_cfi import *
 from RecoLocalTracker.SiPixelRecHits.SiPixelRecHits_cfi import *
 from RecoLocalTracker.SiPixelRecHits.PixelCPEESProducers_cff import *
-pixeltrackerlocalreco = cms.Sequence(siPixelClusters*siPixelRecHits)
-striptrackerlocalreco = cms.Sequence(siStripZeroSuppression*siStripClusters*siStripMatchedRecHits)
-trackerlocalreco = cms.Sequence(pixeltrackerlocalreco*striptrackerlocalreco)
-
+pixeltrackerlocalrecoTask = cms.Task(siPixelClusters,siPixelRecHits)
+pixeltrackerlocalreco = cms.Sequence(pixeltrackerlocalrecoTask)
```

Comment:
This was only needed here internally: sequence not needed any more now, please remove